### PR TITLE
Added more changes

### PR DESCRIPTION
**Pull request by Julias0**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=xr51ALr)

### CODE REVIEW

File: src/app/core/services/accounts.service.ts [hunk 1]
```diff
@@ -19,6 +20,7 @@ export class AccountsService {
     private launchDarklyService: LaunchDarklyService
   ) {}
 
+  @Cacheable()
```
Comment (Julias0) [inline src/app/core/services/accounts.service.ts:23]: accounts dont really change in mobile app, so might as well cache this call.

File: src/app/core/services/org-user-settings.service.ts [hunk 2]
```diff
@@ -33,6 +33,9 @@ export class OrgUserSettingsService {
     return this.apiService.post('/org_user_settings', data);
   }
 
+  @Cacheable({
+    cacheBusterObserver: orgUserSettingsCacheBuster$,
+  })
   getUserSettings(userSettingsId: string) {
```
Comment (Julias0) [inline src/app/core/services/org-user-settings.service.ts:39]: No reason to not cache this

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts [hunk 3]
```diff
@@ -4372,17 +4372,22 @@ export class AddEditExpensePage implements OnInit {
...
-            return this.transactionService.getETxnc({ offset: 0, limit: 100, params }).pipe(
-              map((expenses) => {
-                const expensesArray = expenses as [];
-                return duplicateSets.map((duplicateSet) =>
-                  this.addExpenseDetailsToDuplicateSets(duplicateSet, expensesArray)
-                );
-              })
-            );
+
+            if (duplicateIds.length > 0) {
```
Comment (Julias0) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.ts:4376]: This was making an extra api call if `duplicateIds` is not present, which doesn't make sense.

File: src/app/shared/components/virtual-select/virtual-select-modal/virtual-select-modal.component.ts [hunk 4]
```diff
@@ -164,7 +164,7 @@ export class VirtualSelectModalComponent implements OnInit, AfterViewInit {
         v.isRecentlyUsed = true;
         return v;
       });
-      this.selectableOptions = [...recentlyUsedItemsUpdated, ...filteredOptions];
+      this.selectableOptions = recentlyUsedItemsUpdated.concat(filteredOptions);
```
Comment (Julias0) [inline src/app/shared/components/virtual-select/virtual-select-modal/virtual-select-modal.component.ts:167]: This fixes the duplicated recently used icon from coming

**Review by Dimple16 — APPROVED** [review]

**Review by yksolanki9 — APPROVED** [review]

**Pull request merged**